docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/2.deploy.md

```diff
@@ -0,0 +1,95 @@
+# Deploy full-text index
+
+Nebula Graph full-text indexes are powered by [Elasticserch](https://en.wikipedia.org/wiki/Elasticsearch). Full-text indexes are managed through built-in procedures. They can be created only for variable `STRING` and `FIXED_STRING` properties when the listener cluster and the elastic search cluster are ready.
```

Comment:
Elasticserch==>Elasticsearch

Reply:
elastic search ==> Elasticsearch

Reply:
They can be created for variable `STRING` and `FIXED_STRING` properties only when